linux_os/guide/system/permissions/files/permissions_important_account_files/file_etc_security_opasswd/rule.yml

```diff
@@ -0,0 +1,30 @@
+documentation_complete: true
+
+prodtype: sle12
+
+title: 'Verify Permissions and Ownership of Old Passwords File'
+
+description: '{{{ describe_file_owner(file="/etc/security/opasswd", owner="root") }}}
```

Comment:
That's unfortunate the SLES12 STIG groups all of these three configuration items together, we have a template for each of them and that's something we are always striving for. Having single configuration items per rule, which in past led to many requests to DISA have the STIG requirements split into singles items. I guess it should be fine for now... but leveraging the template would be much better for the long term.

Reply:
Thanks, will keep that in mind.